src/Compilers/VisualBasic/Portable/Binding/LookupResult.vb

```diff
@@ -691,6 +691,12 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                     CompactSymbols(lost)
                     Debug.Assert(_symList.Count > 0)
 
+                    ' As a special case, we allow conflicting enum members imported from
+                    ' metadata If the have the same value, and take the first
```

Comment:
We should not be taking the "first" here, I do not believe the order is guaranteed to be deterministic from compilation to compilation. Perhaps we should choose a member with the "best" name, similar to how we choose the "best" name for overloads elsewhere.